### ISSUE TITLE
NTR: Local anesthetic ineffectiveness (Triggers or responses: drugs or treatments)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9469]
New term request Local anesthetic ineffectiveness (segal_080201172449): Delayed onset or resistance; lidocaine insensitivity

**Comment by pnrobinson** [comment]
@MickeySegal
can you provide more context about this phenotype?

**Comment by pnrobinson** [comment]
@MickeySegal 
The methods in this article state "To assess lidocaine ineffectiveness we developed a non-invasive test based on whether subjects can taste after the tongue is treated with topical lidocaine. "
I do not think this is reliable and it would not be a great HPO term!
There is little other material I can find online.

**Comment by pnrobinson** [comment]
check this

**Comment by pnrobinson** [comment]
@MickeySegal  interesting, I was not aware of this.

Adding new term: 
- ID: HP:6001108
- label: Local anesthetic ineffectiveness
- parents: [Abnormal nervous system electrophysiology (HP:0001311)](https://hpo.jax.org/app/browse/term/HP:0001311)
- pmid: PMID:31723666
- def: Resistance to the effects of local anaesthesia, with delayed onset or failure of local anesthetics to incude numbness or relieve pain.
- comment: This feature has been reported in individuals with Ehlers-Danlos syndrome.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9469
- synonyms:
